scripts/storyteller

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
```

Comment:
sonic-utilities is now only installed as a Python 3 package, so this shebang *should* be fine. However, until we completely remove Python 2 from the image, I suggest changing this to `#!/usr/bin/env python3`, just to be sure.

Reply:
Thanks! Done!